### DISCUSSION TITLE
cuQuantum v23.10.0 and cuQuantum Appliance v23.10

### DISCUSSION THREAD

**Discussion by mtjrider** [NVIDIA/cuQuantum#100] [Announcements]
cuQuantum Appliance v23.10 can be found here:
https://catalog.ngc.nvidia.com/orgs/nvidia/containers/cuquantum-appliance

Documentation and full release notes for all versions are located here:
https://docs.nvidia.com/cuda/cuquantum/latest/index.html

**Comment by erinaldiq** [comment]
Hi @mtjrider,
Nice to see the new release of cuQuantum Appliance!
I could not find the release notes [here](https://docs.nvidia.com/cuda/cuquantum/latest/appliance/release_notes.html)
so I am wondering if the Qiskit Aer simulator integrated with `cusvaer` now supports a noise model https://docs.nvidia.com/cuda/cuquantum/latest/appliance/qiskit.html#selecting-simulator

Thank you!

**Comment by erinaldiq** [comment]
@mtjrider just an update that Neil got the cuQuantum Appliance 23.10 work on Perlmutter in the same way as 23.03.
I am seeing some behavior that I can not explain in terms of how long a SV simulation takes on a single A100-80GB. First of all, is there a way to specify a the device ID of a single GPU when I am running on a node with multiple GPUs?
